keyboards/lovelive9/readme.md

```diff
@@ -0,0 +1,14 @@
+# lovelive9
+
+lovelive9 is 9keys keyboard.
+
+![lovelive9](https://user-images.githubusercontent.com/16838187/53496102-a6fc0700-3ae4-11e9-85e4-1e6cd981b410.jpg)
+
+Keyboard Maintainer: [ginjake](https://github.com/ginjake) [Twitter:@sirojake](https://twitter.com/sirojake)  
+Hardware Supported: Pro Micro
+
+Make example for this keyboard (after setting up your build environment):
+
+    sudo make lovelive9:default
```

Comment:
Using `sudo make` can actually create problems on some systems by changing the ownership of necessary files.

```suggestion
    make lovelive9:default
```